includes/site-health/audit-autoloaded-options/hooks.php

```diff
@@ -26,3 +26,86 @@ function perflab_aao_add_autoloaded_options_test( $tests ) {
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register admin actions for handling autoload enable/disable.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_admin_actions() {
+	add_action( 'admin_action_perflab_aao_update_autoload', 'perflab_aao_handle_update_autoload' );
+}
+add_action( 'admin_init', 'perflab_aao_register_admin_actions' );
+
+/**
+ * Callback for handling disable autoload action.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_handle_update_autoload() {
+	check_admin_referer( 'perflab_aao_update_autoload' );
+
+	if ( ! isset( $_GET['option_name'], $_GET['autoload'] ) ) {
+		wp_die( esc_html__( 'Missing required parameter.', 'performance-lab' ) );
+	}
+
+	$option_name  = sanitize_text_field( wp_unslash( $_GET['option_name'] ) );
+	$autoload     = isset( $_GET['autoload'] ) ? rest_sanitize_boolean( $_GET['autoload'] ) : false;
+	$value_length = isset( $_GET['value_length'] ) ? (int) $_GET['value_length'] : 0;
```

Comment:
Per my above feedback, it's a bit strange we need to pass this here. But more importantly it can get out of date, e.g. when an option is updated. Also, since it's not mandatory, it may be `0` which would be confusing.

I think it would be better to not pass this value to the action URL and instead dynamically retrieve it from the database when displaying the disabled options table.